### ISSUE TITLE
[Accessibility][Name, Role, Value - Filter Dialog]: Name, role and value is not defined for button's available under the 'Filter Dialog'.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2033]
**User Experience** 
Users who rely on AT will face difficulty to interact with the webpage if the name of the controls are not defined properly.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE.
2. Select 'Design Flow' from left navigation.
3. Now, navigate to the 'Filter Dialog'.
4. Navigate to the controls available under the filter dialog such as action sample, actions etc.
5. Observe the issue.

**Actual:** 
The narrator didn't narrate the name,role and value of the control. It only narrates 'group' while navigating through the controls.

**Expected:**
The narrator should announce the name of the control along with their role and state. The role of the control should be list, so that the pane will accessed by up/down arrow and on pressing next tab focus jumps to the 'New trigger' button after which it reach 'New dialog'.

**Suggestions:**
As per accessibility prospective, it is beneficial if the narrator narrates their position so that the user will able to know how much controls are available their.

**MAS Impacted:** [MAS4.1.2](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

**Attachment:**
[Name, role and value is not defined.zip](https://github.com/microsoft/BotFramework-Composer/files/4230879/Name.role.and.value.is.not.defined.zip)

**Comment by corinagum** [comment]
![Name role and value is not correct for the controls available under filter dialog](https://user-images.githubusercontent.com/14900841/75413752-44963c00-58db-11ea-9d69-f5e455989f1e.png)

**Comment by corinagum** [comment]
https://gist.github.com/corinagum/7c74fb84be6c050b6ff52f540bcde1ff

https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html

Based on this spec, the AT should announce how many options are available, but will not read out the entire dropdown list.

**Comment by corinagum** [comment]
Belated update: there wasn't a consensus on how to resolve this problem, so for now we will use stopgap solutions and continue those discussions on how to update the Filter Dialog component with a good a11y experiences. 
@lei9444 thank you for working on this, and my apologies for the late update.

**Comment by beyackle** [comment]
Is this ticket closable? It sounds like the work it refers to is being tracked elsewhere (by those stopgap solutions previously mentioned). Or are we keeping it open so we can refer back to it once these discussions have happened?

**Comment by beyackle** [comment]
Okay, in this issue as written there is still one repro-able bug, and that is that the column beneath "Filter Dialog" is not navigable with the arrow keys, just with Tab. I'll take that on.

**Comment by beyackle** [comment]
And now that I look at it again, PR #2914 will address that exact issue.

**Comment by ashish315** [comment]
As checked now the narrator announces the name of the control and will accessible with up/down key so HCL is closing the issue.
